tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/JSDesugaring.scala

```diff
@@ -1540,8 +1540,15 @@ private[emitter] class JSDesugaring(internalOptions: InternalOptions) {
         case Apply(receiver, method, args) =>
           val newReceiver = transformExpr(receiver)
           val newArgs = args map transformExpr
+
+          /* If the receiver is maybe a hijacked class instance, and there
+           * exists a hijacked method helper for the method, use it. Methods
+           * that do not have helpers are
+           * - Reflective proxies
+           * - Methods of RuntimeLong that are not also in java.lang.Long
+           */
           if (isMaybeHijackedClass(receiver.tpe) &&
-              !Definitions.isReflProxyName(method.name)) {
+              hijackedClassMethodToHelperName.contains(method.name)) {
```

Comment:
Just to check my understanding: This change is necessary, since the optimizer now emits `RuntimeLongs` where before it emitted primitive `Long`s, right?